web_widget_section_and_note_text_scrollable/static/src/product_label_section_and_note_field_scrollable.scss

```
@@ -0,0 +1,30 @@
+/*
+-    Copyright 2025 Camptocamp SA (https://www.camptocamp.com).
+-    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+-*/
+
+/* 1) Override the inline heights coming from the widget */
+.o_field_product_label_section_and_note_field,
+.o_field_product_label_section_and_note_cell,
+.o_field_product_label_section_and_note_cell textarea {
+    height: auto !important;
+}
```

I'm a (s)css noob, but I think `height` is anyway redefined in `o_field_product_label_section_and_note_cell` and `o_field_product_label_section_and_note_cell textarea` below.

```suggestion
.o_field_product_label_section_and_note_field {
    height: auto !important;
}
```